src/Microsoft.VisualStudio.ProjectSystem.Managed/Telemetry/ILanguageServiceTelemetryService.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using Microsoft.VisualStudio.Composition;
+using Microsoft.VisualStudio.ProjectSystem;
+
+namespace Microsoft.VisualStudio.Telemetry
+{
+    [ProjectSystemContract(ProjectSystemContractScope.Global, ProjectSystemContractProvider.Private, Cardinality = ImportCardinality.ExactlyOne)]
```

Comment:
The implemetnation of this interface imports `UnconfiguredProject`, so this interface should declare itself as being in the `UnconfiguredProject` scope.

Reply:
Which implementation class are you referring to? I don't see the UnconfiguredProject import.

---
In reply to: [566549849](https://github.com/dotnet/project-system/pull/6880#discussion_r566549849) [](ancestors = 566549849)

Reply:
I got your point from the comment on the PackageRestoreConfiguredInputDataSource. Investigating this change.

---
In reply to: [566994525](https://github.com/dotnet/project-system/pull/6880#discussion_r566994525) [](ancestors = 566994525,566549849)